src/graph/ConfigExecutor.cpp

```diff
@@ -0,0 +1,318 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/ConfigExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+ConfigExecutor::ConfigExecutor(Sentence *sentence,
+                               ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<ConfigSentence*>(sentence);
+}
+
+Status ConfigExecutor::prepare() {
+    configItem_ = sentence_->configItem();
+    return Status::OK();
+}
+
+void ConfigExecutor::execute() {
+    auto showType = sentence_->subType();
+    switch (showType) {
+        case ConfigSentence::SubType::kShow:
+            showVariables();
+            break;
+        case ConfigSentence::SubType::kSet:
+            setVariables();
+            break;
+        case ConfigSentence::SubType::kGet:
+            getVariables();
+            break;
+        case ConfigSentence::SubType::kDeclare:
+            declareVariables();
+            break;
+        case ConfigSentence::SubType::kUnknown:
+            onError_(Status::Error("Type unknown"));
+            break;
+    }
+}
+
+void ConfigExecutor::showVariables() {
+    std::string space;
+    std::string module;
+    if (configItem_ != nullptr) {
+        if (configItem_->getSpace() != nullptr) {
+            space = *configItem_->getSpace();
+        }
+        if (configItem_->getModule() != nullptr) {
+            module = *configItem_->getModule();
+        }
+    }
+
```

Comment:
maybe need return directly when space and module is empty string

Reply:
When space and module is empty, it will show all configs just like "show variables" in mysql.

Reply:
so can there generate a default response directly when parameters are empty?

Reply:
For now, there is no default response. Only if no configs was set before indeed, it get empty result.